src/fairseq2/nn/utils/module.py

```diff
@@ -570,3 +571,44 @@ def get_module_size(module: Module) -> ModuleSizeInfo:
         info.total_size_bytes += size_bytes
 
     return info
+
+
+def normalize_truncate(
+    tensor: Tensor,
+    *,
+    mean: float = 0.0,
+    std: float = 1.0,
+    a: float = -2.0,
+    b: float = 2.0,
+) -> None:
+    def _norm_cdf(x: float) -> float:
+        # Computes standard normal cumulative distribution function
+        return (1.0 + math.erf(x / math.sqrt(2.0))) / 2.0
+
+    lower = _norm_cdf((a - mean) / std)
+    upper = _norm_cdf((b - mean) / std)
+
+    tensor.uniform_(2 * lower - 1, 2 * upper - 1)
+    tensor.erfinv_()
+
+    tensor.mul_(std * math.sqrt(2.0))
+    tensor.add_(mean)
+
+    tensor.clamp_(min=a, max=b)
+
+
+def init_truncated_uniforma_weights_and_bias(
```

Comment:
I would prefer this function to be within JEPA's factory.py instead of this file which is meant typically for much more generic module helper functions.